src/test/java/caculator/CaculatorTest.java

```diff
@@ -0,0 +1,132 @@
+package caculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import static org.assertj.core.api.Assertions.*;
+
+public class CaculatorTest {
+
+    @Test
+    public void splitAndSum_null_또는_빈문자() throws Exception {
+        int result = splitString(null);
+        assertThat(result).isEqualTo(0);
```

Comment:
```suggestion
        Calculator calculator = new Calculator()   // 예시
        int result = calculator.calculate(null);        // 예시
        assertThat(result).isEqualTo(0);
```
테스트 영역에는 테스트 대상을 실행하고 검증하는 공간입니다.
그러므로 계산 로직은 test 패키지가 아닌 프로덕션 영역인 main 패키지에 위치하는 것이 맞습니다.

계산기 객체를 만들고 계산기를 테스트하는 방향으로 변경하면 좋겠습니다.